snippets/base/models.py

```diff
@@ -577,11 +578,26 @@ class Meta:
 
 class TargetedCountry(CachingMixin, models.Model):
     code = CountryField('Geolocation Country', unique=True)
+    name = models.CharField(max_length=100)
+    priority = models.BooleanField(default=False)
 
     objects = CachingManager()
 
     def __unicode__(self):
```

Comment:
Nice. No point adding this now because the codebase is not python3 compatible anyway, but I'll add it as a reminder.